mmv1/third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -170,6 +170,14 @@ func resourceComputeInstanceGroupManager() *schema.Resource {
 				Description: `The target number of running instances for this managed instance group. This value should always be explicitly set unless this resource is attached to an autoscaler, in which case it should never be set. Defaults to 0.`,
 			},
 
+			"list_managed_instances_results": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice([]string{"PAGELESS", "PAGINATED", ""}, false),
+				DiffSuppressFunc: emptyOrDefaultStringSuppress("PAGELESS"),
```

Comment:
A DSF doesn't quite do what we want- it'll keep values in state if it returns true, ignoring the value in the user's config. We should set a `Default` value instead, to the API default. Will the API return the default value explicitly for all IGMs/RIGMS, including older ones that predate the feature?

Reply:
Yes, the API will return the PAGELESS default for old MIGs. I see, the default makes sense. PTAL now